_data/navbars/maintenance.yml

```diff
@@ -11,6 +11,11 @@ section:
     path: /maintenance/openshift-upgrade
   - title: OpenStack
     path: /maintenance/openstack-upgrade
+- title: Deploy image options
```

Comment:
Do we intend to have addition items in this sub-heading? there's not a ton of point to have a heading with a single entry in it unless we expect there to be additional articles here in the near future.

Reply:
Yeah this is where private registry documentation should go.